features/financial_assistance/step_definitions/review_applications.rb

```diff
@@ -47,3 +47,27 @@
   # Submit Application
   find("#application_medicaid_terms").click
 end
+
+Given(/the non applicant citizen status feature is enabled/) do
+  EnrollRegistry[:non_applicant_citizen_status].feature.stub(:is_enabled).and_return(true)
+end
+
+Given(/the non applicant citizen status feature is disabled/) do
+  EnrollRegistry[:non_applicant_citizen_status].feature.stub(:is_enabled).and_return(false)
+end
+
+Given(/a family has a non applicant member/) do
+  @family = user.primary_family
+  FactoryBot.create(:family_member, family: @family)
+  applicant = FactoryBot.create(:financial_assistance_applicant, is_applying_coverage: false, application: application, dob: Date.new(2000,1,1), gender: "male", citizen_status: "not_lawfully_present_in_us")
+  applicant.relationships.create(kind:"spouse", applicant_id: applicant.id, relative_id: application.applicants.first.id)
+end
+
+Then(/the user will see the nonapplicant citizen status as N\/A/) do
+  expect(page).to_not have_content("Not lawfully present in US")
```

Comment:
Need to replace these hard-coded english strings with Page objects that reference a css class/attribute

Reply:
make sense..